bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: make sure pull secrets are created on the clusters if they are set

## Description


https://issues.redhat.com/browse/MGDSTRM-2004

## Verification Steps


* Make sure you have a cluster setup and checkout the branch
* Make sure you copy/paste the pull secret content to `secrets/image-pull.dockerconfigjson` file
* Run the service. Once the syncset is updated, you can login to the cluster and you should see a pull secret called `rhoas-image-pull-secret` is created in the `redhat-managed-kafka-operator` namespace

## Type of change


## Checklist (Definition of Done)
